src/build.jl

```diff
@@ -9,7 +9,19 @@ latexengine!(eng::LaTeXEngine) = ACTIVE_LATEX_ENGINE[] = eng
 
 _engine_cmd(eng::LaTeXEngine) = `$(lowercase(string(eng)))`
 
-_latex_cmd(file::String, eng::LaTeXEngine, flags) = `$(_engine_cmd(eng)) $flags $file`
+"""
+    $SIGNATURES
+
+Return a `Cmd` object for running LaTeX on `filename`, which can be a relative
+or an absolute path (recommended). Output files end up in the same directory.
+"""
+function _latex_cmd(filename::String, eng::LaTeXEngine, flags)
```

Comment:
So the reason that I always (I think) output the latex files to the current directory and then move them in Julia instead of using `output-directory` is due to issues like https://tex.stackexchange.com/questions/109331/using-pgfplots-and-gnuplot-and-compiling-with-output-directory. I found no way around it.

Reply:
I see, sorry I did not know about this. I changed the tests to detect this (previously it passed), and also fixed it.